.github/workflows/build-flink-connectors.yaml

```diff
@@ -30,3 +30,5 @@ jobs:
         uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
       - name: Build with Gradle
         run: ./gradlew build
+      - name: Scan with Gradle
+        run: ./gradlew build --scan
```

Comment:
Just merge them as one step.

Reply:
OK, so now I'm running the scan with every build :)